<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_read_Image">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_Image">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startTimer">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Timer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_endTimer">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>End Timer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_timerDocu">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_matrix">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_testTimer">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2000ms Test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>541</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_oriImage">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_finalImage">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>430</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>After Processing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_userMatrix">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>320</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_conMatrix">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>450</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_matTest">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_matTest_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>480</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_matTest_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_matTest_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_matTest_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>510</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_matTest_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_matTest_7">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>540</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_matTest_8">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_matTest_9">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>540</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clearMatrix">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_width">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>475</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_length">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>475</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_char">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>480</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pixel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>470</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Pixel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Grayscale">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>470</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Binary">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>470</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Light">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>470</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Light</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_timerDocuConvolution">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>530</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_timeConvolution">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>530</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Convolution">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>520</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Convolution</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
